prow/cmd/deck/static/script.js

```diff
@@ -528,6 +528,19 @@ function redraw(fz) {
             r.appendChild(createTextCell(""));
             r.appendChild(createTextCell(""));
         }
+        if (spyglass) {
+            if (build.url.startsWith("https://k8s-gubernator.appspot.com/build/")) {
+                const icon = createIcon("visibility", "View in Spyglass");
+                icon.href = build.url.replace("https://k8s-gubernator.appspot.com/build/",
+                    "https://prow.k8s.io/view/gcs/");
```

Comment:
What about jobs currently mid-run where they have not published to GCS?

Reply:
The current behavior is if there's no Gubernator link, there's no Spyglass link either (shown in the screenshot at the top).

Reply:
the running ones (the yellow ones) will have gubernator link.. @paulangton I remember spyglass also can handle pod logs right?

Reply:
yes it can, check out https://prow.k8s.io/view/prowjob/pull-kubernetes-e2e-gce/52729